chain/jsonrpc/src/lib.rs

```diff
@@ -29,6 +27,19 @@ use crate::message::{Request, RpcError};
 pub mod client;
 mod message;
 
+macro_rules! select { // replace `::futures_util` with `::futures03` as the crate path
+    ($($tokens:tt)*) => {
+        futures03::inner_select::select! {
```

Comment:
How does this work? https://github.com/rust-lang-nursery/futures-rs/blob/master/futures-select-macro/src/lib.rs does not provide documentation :(

Reply:
@nearmax I learned this trick here: https://github.com/rust-lang-nursery/futures-rs/blob/c30adf513b9eea35ab385c0797210c77986fc82f/futures/src/lib.rs#L503-L510